src/olympia/users/tests/test_admin.py

```diff
@@ -48,7 +48,7 @@ def setUp(self):
         )
 
     def test_search_by_email_simple(self):
-        user = user_factory(email='someone@mozilla.com')
+        user = user_factory(email='someone@mozilla.com', username='test')
```

Comment:
username contains a uuid which has roughly an 11% chance of matching the pk for a user we expect "NOT" to be in the document. adding the static username brings that chance down to zero.

Reply:
Ah, makes sense, I forgot about the fact that the default username was generated.

I suggested a different fix on slack which relied on checking for the email presence, but that's fine as well.

Reply:
Ex: https://github.com/mozilla/addons-server/actions/runs/9415386157/job/25936381894

Reply:
Can you share the fix here? Cannot find on slack.

Reply:
```diff
diff --git a/src/olympia/users/tests/test_admin.py b/src/olympia/users/tests/test_admin.py
index fc6514760d..f58e9c0356 100644
--- a/src/olympia/users/tests/test_admin.py
+++ b/src/olympia/users/tests/test_admin.py
@@ -121,9 +121,10 @@ class TestUserAdmin(TestCase):
         )
         assert response.status_code == 200
         doc = pq(response.content)
-        assert str(user.pk) in doc('#result_list').text()
-        assert str(another_user.pk) in doc('#result_list').text()
-        assert str(self.user.pk) not in doc('#result_list').text()
+        # If found, the user pk will be present followed by a colon.
+        assert f'{user.pk}:' in doc('#result_list').text()
+        assert f'{another_user.pk}:' in doc('#result_list').text()
+        assert f'{self.user.pk}:' not in doc('#result_list').text()
 
     def test_search_for_multiple_user_ids(self):
         """Test the optimization when just searching for matching ids."""
```

Essentially searching for `{pk}:` (with the colon) instead of just the pk in the response. Crude, but it works. Then copy to other tests doing the same thing.